src/main/java/io/personium/core/PersoniumMeasurmentLog.java

```diff
@@ -0,0 +1,113 @@
+/**
+ * personium.io
+ * Copyright 2014-2019 FUJITSU LIMITED
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.personium.core;
+
+import java.text.MessageFormat;
+
+/**
+ * Log message with measurement creation class.
+ */
+public final class PersoniumMeasurmentLog extends PersoniumCoreLog {
```

Comment:
Class name "PersoniumMeasurmentLog " sounds a bit unclear to me since it does not state what is measured. This class only measures elapsed time, so what about "ElapsedTimeLog" ?

Reply:
Thank you for your comment. I renamed PersoniumMeasurmentLog to ElapsedTimeLog.